### PR TITLE
Nucleo f411re support

### PR DESCRIPTION
**Pull request by FredPraca**
Bareboard runtime for Nucleo-F411RE using STM32F411re

### CODE REVIEW

File: .gitreview [hunk 1]
```diff
@@ -3,4 +3,4 @@ host = review.eu.adacore.com
 port = 29418
 project = bb-runtimes
 defaultbranch = community-2020
-defaultremote = origin
+xsdefaultremote = origin
```
Comment (Fabien-Chouteau) [inline .gitreview:6]: Why change this line?

**Review by Fabien-Chouteau — COMMENTED** [review]
Thank you @FredPraca !

The code looks good to me. I just have one question.

**Comment by Fabien-Chouteau** [comment]
Also you have to rebase your changes on our current branch:
```
git remote add upstream https://github.com/AdaCore/bb-runtimes.git
git pull --rebase --autostash upstream community-2020
```

File: .gitreview (hunk 1 above)
Comment (FredPraca) [reply .gitreview:6]: Looks like ctrl+x ctrl+s in Emacs for saving :laughing: 
But I'm not sure, I don't remember changing it myself, could it be Gnatstudio doing something like that ?

**Comment by FredPraca** [comment]
When rebasing, commits on Nucleo F401re will be included because of a wrong manipulation I made on local branches... Does it matter ?

**Comment by FredPraca** [comment]
Rebasing broke the whole thing, so I close the request and will try to fix it up

**Pull request closed without merging**